piccolo/query/methods/select.py

```diff
@@ -46,6 +72,71 @@ def get_select_string(self, engine_type: str, just_alias=False) -> str:
         return f"COUNT({column_name}) AS count"
 
 
+class Max(Selectable):
+    """
+    MAX() SQL function.
+
+    await Band.select(Max(Band.popularity)).run() or with aliases
+    await Band.select(Max(Band.popularity, alias="popularity_max")).run()
+    """
+
+    def __init__(self, column: Column, alias: str = "max"):
+        self.column = column
+        self.alias = alias
+
+    def as_alias(self, alias: str):  # pragma: no cover
+        self.alias = alias
+
+    def get_select_string(self, engine_type: str, just_alias=False) -> str:
+        column_name = self.column._meta.get_full_name(just_alias=just_alias)
+        return f"MAX({column_name}) AS {self.alias}"
+
+
+class Min(Selectable):
+    """
+    MIN() SQL function.
+
+    await Band.select(Min(Band.popularity)).run() or with aliases
+    await Band.select(Min(Band.popularity, alias="popularity_min")).run()
+    """
+
+    def __init__(self, column: Column, alias: str = "min"):
+        self.column = column
+        self.alias = alias
+
+    def as_alias(self, alias: str):  # pragma: no cover
```

Comment:
@sinisaos Sorry to jump in again, but I think you can add unit tests for the `as_alias` method so you don't have to exclude them from test coverage using `#  pragma: no cover`

Reply:
@aminalaee I agree with you. I changed the code to actually use `as_alias` method and the test now don't need `# pragma: no cover`.

```python
class Sum(Selectable):
    """
    SUM() SQL function. Column type must be numeric to run the query.

    await Band.select(Sum(Band.popularity)).run() or with aliases
    await Band.select(Sum(Band.popularity).as_alias("popularity_sum")).run()
    """

    def __init__(self, column: Column, alias: str = "sum"):
        self.column = column
        self.alias = alias

    def as_alias(self, alias: str) -> Sum:
        column = copy.deepcopy(self)
        column.alias = alias
        return column

    def get_select_string(self, engine_type: str, just_alias=False) -> str:
        if self.column.value_type in (int, decimal.Decimal, float):
            column_name = self.column._meta.get_full_name(
                just_alias=just_alias
            )
        else:
            raise ValueError("Column type must be numeric to run the query.")
        return f"SUM({column_name}) AS {self.alias}"
```
The query with aliases is now like this `await Band.select(Sum(Band.popularity).as_alias("popularity_sum")).run()`.  What do you think? Is this better or are we going back to the previous code?

Reply:
@sinisaos The `as_alias` method can be this (no need to do the copy):

```python
    def as_alias(self, alias: str) -> Sum:
        self.alias = alias
        return self
```

I think it's worth having the `as_alias` because that's how it works with other parts of the Piccolo code base, for example `await User.select(User.name.as_alias('title')).run()`.

I expect most people will use `Sum(MyTable.column, alias="foo")`, but having `as_alias` is a nice to have, to keep things consistent.

Reply:
@dantownsend That's great. Thanks. Now we can make query in two ways `await Band.select(Sum(Band.popularity, alias="popularity_sum")).run()` and `await Band.select(Sum(Band.popularity).as_alias("popularity_sum")).run()` and coverage is fine. I'll add this to docs.